src/Silk.Shared/Constants/StringConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Silk.Shared.Constants
+{
+    public static class StringConstants
+    {
+        public const string Version = "1.5.2-alpha";
```

Comment:
Following up on my previous comment about Versionning, this is something you can very much set in package info, and get using `Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion`. This would eliminate the need to set it twice (if you set it in package info too).

Reply:
Will look into it; it's also used for things like our HTTP headers, so, it may remain a constant.